.github/workflows/IJ.yml

```diff
@@ -50,8 +45,7 @@ jobs:
         run: chmod +x gradlew
       - name: Build with Gradle
         run: >
-          ./gradlew runPluginVerifier
-          -PideaVersion=IC-2022.3
+          ./gradlew verifyPlugin
```

Comment:
this step is not run because of the` if: false` in line 32 above. please remove it.
also, the entire `validate` step is redundant with the build workflow.